third_party/terraform/resources/resource_google_project.go

```diff
@@ -227,7 +227,13 @@ func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error
 	d.SetId(pid)
 
 	// Wait for the operation to complete
-	waitErr := resourceManagerOperationWait(config.clientResourceManager, op, "project to create")
+  opV1 := make(map[string]interface{})
+  waitErr := Convert(op, opV1)
+  if waitErr != nil {
+    return waitErr
+  }
+
+	waitErr = resourceManagerOperationWaitTime(config, opV1, "", "creating folder", int(d.Timeout(schema.TimeoutCreate).Minutes()))
```

Comment:
Do we need the `project` on any generated operation waiters? I'm assuming it's stored in the operation `name` even for APIs that will use it later like compute.

Reply:
ummm... good question. I was partially assuming we would need it in the future, since the operations needed it originally.